### PR TITLE
Moving SQL migration scripts and adding README

### PR DESCRIPTION
**Pull request by matzew**
some org. changes. Moving the migration related content into its own folder.

Adding a README for instructions as well

### CODE REVIEW

**Comment by matzew** [comment]
@edewit @sebastienblanc  can you guys review?

File: databases/migration/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+### MySQL
+
+Inside of the database execute these two scripts:
+
+* `keycloak-migration-mysql.sql`
+* `ups-migration-mysql.sql`
+
+### PostgreSQL
+
+Inside of the database execute these two scripts:
```
Comment (sebastienblanc) [inline databases/migration/README.md]: Should mention that the order is important (first KC and then UPS or is the order not relevant ?

Comment (matzew) [inline databases/migration/README.md]: is irrelevant

**Comment by edewit** [comment]
:+1:

**Comment by sebastienblanc** [comment]
Landed 1.0.x f0d1ce60629b815cf2e62b822e202c5a82e9b603 and master 47a11e3db5898bdded0020b28621b530db819f67

**Pull request closed without merging**